src/background/telemetry/metrics.js

```diff
@@ -357,20 +352,7 @@ export default class Metrics {
     this.storage.active_daily_velocity = active_daily_velocity;
     this.saveStorage(this.storage);
 
-    const daily = this._timeToExpired('active', 'daily');
-    if (daily <= 0) {
-      this._sendReq('active', ['daily']);
-    }
-
-    const weekly = this._timeToExpired('active', 'weekly');
-    if (weekly <= 0) {
-      this._sendReq('active', ['weekly']);
-    }
-
-    const monthly = this._timeToExpired('active', 'monthly');
-    if (monthly <= 0) {
-      this._sendReq('active', ['monthly']);
-    }
+    this._sendReq('active', ['daily', 'weekly', 'monthly']);
```

Comment:
The `_sendReq` checks each frequency if the time expired... So I cleaned this up to match L386